config/config.go

```diff
@@ -287,7 +288,8 @@ func (c *Config) extend(extensionConfig Config) {
 			// Rule doesn't exist, add it to the config.
 			c.Rules[ruleID] = baseRule
 			for _, k := range baseRule.Keywords {
-				c.Keywords[k] = struct{}{}
+				// Craig test this - extend_with_new_rule
+				c.Keywords[strings.ToLower(k)] = struct{}{}
```

Comment:
This change is covered by https://github.com/gitleaks/gitleaks/pull/1633/files#diff-373f5578b8f362a364b50b0d01919a57da64c128547ca10d49df8ae422237c82R410